tests/unit/linking/test_link.py

```diff
@@ -186,16 +189,17 @@ def test_compare_identifier_with_suffix(self):
             evaluators=[
                 {"feature": "IDENTIFIER:MR", "func": "func:recordlinker.linking.matchers.compare_probabilistic_fuzzy_match"},
             ],
-            rule="func:recordlinker.linking.matchers.rule_probabilistic_match",
-            kwargs={"log_odds": {"IDENTIFIER": 0.35}, "true_match_threshold": 0.3},
+            rule="func:recordlinker.linking.matchers.rule_probabilistic_sum",
+            possible_match_window=(0.8, 0.925),
+            kwargs={"log_odds": {"IDENTIFIER": 0.35}},
         )
 
         #should pass as MR is the same for both
-        assert link.compare(rec, pat, algorithm_pass) is True
+        assert link.compare(rec, pat, algorithm_pass) == 0.35
```

Comment:
Hm. this is beyond the scope of this PR but I'm wondering if we should be treating all identifiers equally. It makes sense to me to give full points if the patient agrees with any identifiers of the same type in a record since you can have multiple MRNs, driver's licenses, etc. However, SSN should not be different because it's a national ID. I'm wondering if we should treat SSN differently.

Reply:
I do think this is something worth investigating. I think we'll have to base this heavily on what happens with Montana  and creating stuff for them to run. We'd have to do profiling around SSN first anyway since we didn't even have that data nicely presented before.